lib/twine/formatters/abstract.rb

```diff
@@ -3,17 +3,28 @@
 module Twine
   module Formatters
     class Abstract
-      attr_reader :strings
-      attr_reader :options
+      attr_accessor :strings
+      attr_accessor :options
 
-      def self.can_handle_directory?(path)
-        return false
-      end
-
-      def initialize(strings, options)
+      def initialize(strings = StringsFile.new, options = {})
```

Comment:
The common use case is to pass in the strings and options, no? Only unit tests might want to use the default arguments here of a new StringsFile and an empty options dictionary. As that is the case, I prefer the old method signature. The unit tests can pass these default arguments in themselves.